### PR TITLE
Fortinet's new module for fortios_switch_controller_qos_qos_policy

### PR DESCRIPTION
**Pull request by migumun**
##### SUMMARY
 
Fortinet is adding Ansible support for FortiOS and FortiGate products. This module follows the same structure, guidelines and ideas given in previous approved module for a parallel feature of FortiGate (user device): https://github.com/ansible/ansible/pull/56447
In this case we are providing a different functionality: "Fortios FortiSwitch QoS policy".
 
Please note that this will be part of other modules to come for FortiGate, including different functionalities: system, wireless-controller, firewall, webfilter, ips, web-proxy, wanopt, application, dlp spamfilter, log, vpn, certificate, user, dnsfilter, antivirus, report, waf, authentication, switch controller, endpoint-control and router. We plan to follow the same style, structure and usage as in the previous module in order to make it easier to comply with Ansible guidelines.
 
This PR has been updated according to suggestions and code reviews given in previous submissions of Fortinet modules.

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

fortios_switch_controller_qos_qos_policy

##### ANSIBLE VERSION

```
ansible 2.8.2
python version = 3.7.3 (default, Apr  3 2019, 05:39:12) [GCC 8.3.0]
```

### CODE REVIEW

File: lib/ansible/modules/network/fortios/fortios_switch_controller_qos_qos_policy.py [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+    return not is_successful_status(resp), \
+        resp['status'] == "success", \
+        resp
+
+
+def main():
+    fields = {
+        "host": {"required": False, "type": "str"},
+        "username": {"required": False, "type": "str"},
+        "password": {"required": False, "type": "str", "no_log": True},
```
Comment (rohitthakur2590) [inline lib/ansible/modules/network/fortios/fortios_switch_controller_qos_qos_policy.py]: default value is missing

File: lib/ansible/modules/network/fortios/fortios_switch_controller_qos_qos_policy.py [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+        description:
+            - Indicates if the requests towards FortiGate must use HTTPS protocol.
+        type: bool
+        default: true
+    ssl_verify:
+        description:
+            - Ensures FortiGate certificate must be verified by a proper CA.
+        type: bool
+        default: true
+    state:
```
Comment (rohitthakur2590) [inline lib/ansible/modules/network/fortios/fortios_switch_controller_qos_qos_policy.py:75]: 'required' is missing

File: lib/ansible/modules/network/fortios/fortios_switch_controller_qos_qos_policy.py (hunk 1 above)
Comment (migumun) [reply lib/ansible/modules/network/fortios/fortios_switch_controller_qos_qos_policy.py]: I have added a new commit fixing this

File: lib/ansible/modules/network/fortios/fortios_switch_controller_qos_qos_policy.py (hunk 2 above)
Comment (migumun) [reply lib/ansible/modules/network/fortios/fortios_switch_controller_qos_qos_policy.py:75]: I have added a new commit fixing this

**Pull request merged**